sch_conditions.c

```diff
@@ -25,19 +25,14 @@
 #include <inttypes.h>
 #define APTERYX_XML_LIBXML2
 
-
 typedef struct _cond_result
 {
     bool result;
+    char *path;
```

Comment:
Is path actually used anywhere?

Reply:
Now that you mention it, probably not. I'll remove it.

Reply:
Actually, it is used in the tree traversal looking for the path. So no change required for this comment